.github/mergify.yml

```diff
@@ -0,0 +1,43 @@
+pull_request_rules:
+- actions:
+    merge:
+      strict: smart
```

Comment:
Will this automate the "update branch" step too?

Reply:
Answering myself: it will. 

Did you consider `smart+fasttrack` ?

Reply:
yeah, but maybe we need at some point merge pr's in some predetermined order? well, thinking twice right now, we could get the order via the `merge me` label

Reply:
`smart` will put new PRs at the back of the "notify me" queue. 
`smart+fasttrack` will merge new PRs straight away if they are up-to-date (and pass all signals). Otherwise they will go back to the end of the queue. 

Since `smart+fasttrack` seems slightly more efficient, I would use that